regression/pages/studio/privacy_policy.py

```diff
@@ -12,6 +12,8 @@ class PrivacyPolicy(PageObject):
     url = LOGIN_BASE_URL + '/edx-privacy-policy'
 
     def is_browser_on_page(self):
-        return "As used in this Privacy Policy" in self.q(
-            css='.field-page-body'
-        ).text[0]
+        # Temporary condition for Acquia issue
```

Comment:
This would be a good fix if it is needed.  Two thoughts:
1. I just remembered that yesterday @jaebradley told me that the issue was fixed in Stage, although it persists in other Acquia test environments like Acceptance.  The e2e tests are run against Stage, so we can probably do a simple revert without this change.  @jaebradley: Can you confirm?
2. If I am wrong and the issue is still happening in Stage, please create a clean-up JIRA ticket that is blocked on the Acquia fix and then update these comments to read:
```
# TODO: LEARNER-XXXX: Clean-up temporary condition for Acquia issue
```

Reply:
@robrap also confirming that all non-prod environments (like `acceptance`) no longer have the intermittent 404 issue